kong/init.lua

```diff
@@ -735,7 +739,9 @@ function Kong.handle_error()
   ctx.KONG_UNEXPECTED = true
 
   if not ctx.plugins then
-    runloop.update_plugins_iterator()
+
+    local timeout = runloop.get_rebuild_timeout()
+    runloop.rebuild_plugins_iterator(timeout) -- ignore error
```

Comment:
Same here.

Reply:
Answered in other thread, closing this one to keep the conversation focused.